tests/basic/storage_pool_test.go

```diff
@@ -5188,3 +5190,192 @@ var _ = Describe("{ResizePoolDrivesInDifferentSize}", func() {
+			// store the new label that is updated
+		})
+
+		// Let the expand complete
+		var expectedSize uint64
+		var expectedSizeWithJournal uint64
+		var contextToDel *scheduler.Context
+
+		labelBeforeExpand := poolToBeResized.Labels
+		stepLog = "Calculate expected pool size and trigger pool resize"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			expectedSize = poolToBeResized.TotalSize * 2 / units.GiB
+			isjournal, err := isJournalEnabled()
+			log.FailOnError(err, "Failed to check if Journal enabled")
+			// To-Do Need to handle the case for multiple pools
+			expectedSizeWithJournal = expectedSize
+			if isjournal {
+				expectedSizeWithJournal = expectedSizeWithJournal - 3
+			}
+			log.InfoD("Current Size of the pool %s is %d", poolIDToResize, poolToBeResized.TotalSize/units.GiB)
+			// Delete the Volume that was ~190G before the pool expand begins
+			// Iterate through the contexts, get the volumes and then get the matching ID
+		gotContext:
+			for _, l := range contexts {
+				vols, err := Inst().S.GetVolumes(l)
+				dash.VerifyFatal(err, nil, "Verify if able to get the app for the volume that is filled approx 200G")
+				for _, vol := range vols {
+					if vol.ID == volID {
+						contextToDel = l
+						break gotContext
+					}
+				}
+			}
+			err = Inst().V.ExpandPool(poolIDToResize, api.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, expectedSize)
+			dash.VerifyFatal(err, nil, "Pool expansion init successful?")
+			// Destroy the context
+			err = Inst().S.Destroy(contextToDel, nil)
+			dash.VerifyFatal(err, nil, "Verify the successful delete context of the volume which had ~190 G usage")
+			log.InfoD("Going to delete the volume, by deletion of Namespace")
+			TearDownContext(contextToDel, map[string]bool{
+				SkipClusterScopedObjects:                    false,
+				scheduler.OptionsWaitForResourceLeakCleanup: true,
+				scheduler.OptionsWaitForDestroy:             true,
+			})
+			dash.VerifyFatal(err, nil, "Verify the successful delete of the volume which had ~190 G usage")
+			resizeErr := waitForPoolToBeResized(expectedSize, poolIDToResize, isjournal)
+			dash.VerifyFatal(resizeErr, nil, fmt.Sprintf("Expected new size to be '%d' or '%d'", expectedSize, expectedSizeWithJournal))
+		})
+		// Make sure to remove the deleted context and validate the other apps
+		for _, l := range contexts {
+			if l.App.Key != contextToDel.App.Key {
+				newContexts = append(newContexts, l)
+			}
+		}
+		stepLog = "Ensure that new pool has been expanded to the expected size and also check the pool expand alert"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			ValidateApplications(newContexts)
+			resizedPool, err := GetStoragePoolByUUID(poolIDToResize)
+			log.FailOnError(err, fmt.Sprintf(" Failed to get pool using UUID %s", poolIDToResize))
+			newPoolSize := resizedPool.TotalSize / units.GiB
+			isExpansionSuccess := false
+			if newPoolSize == expectedSize || newPoolSize == expectedSizeWithJournal {
+				isExpansionSuccess = true
```

Comment:
change this condition to just newPoolSize >= expectedSizeWithJournal

Reply:
done